src/electron-app.js

```diff
@@ -1,36 +1,69 @@
-const electron = require('electron');
-const app = electron.app;  // Module to control application life.
-const ipc = electron.ipcMain;
-const BrowserWindow = require('browser-window');  // Module to create native browser window.
-const localShortcut = require('electron-localshortcut');
-
-// Keep a global reference of the window object, if you don't, the window will
-// be closed automatically when the javascript object is GCed.
+import { app, ipcMain as ipc, BrowserWindow } from 'electron';
+import localShortcut from 'electron-localshortcut';
+import GithubReleases from 'electron-gh-releases';
+
+const updater = new GithubReleases({
+  repo: 'james-proxy/james',
+  currentVersion: app.getVersion()
+});
```

Comment:
Should put these into [`constants.js`](https://github.com/james-proxy/james/blob/master/src/constants.js), especially for version, as it's used for raven as well.